crest/Assets/Crest/Crest/Shaders/Underwater/Resources/UnderwaterEffect.shader

```diff
@@ -112,12 +114,102 @@ Shader "Hidden/Crest/Underwater/Underwater Effect"
 					float3 scenePos = _WorldSpaceCameraPos - view * sceneZ / dot(unity_CameraToWorld._m02_m12_m22, -view);
 					const float3 lightDir = _WorldSpaceLightPos0.xyz;
 					const half3 lightCol = _LightColor0;
-					sceneColour = ApplyUnderwaterEffect(scenePos, sceneColour, lightCol, lightDir, rawDepth, sceneZ, view, isOceanSurface);
+					sceneColour = ApplyUnderwaterEffect(_AmbientLighting, scenePos, sceneColour, lightCol, lightDir, rawDepth, sceneZ, view, isOceanSurface);
 				}
 
 				return half4(wt * sceneColour, 1.0);
 			}
 			ENDHLSL
 		}
+
+		Pass
```

Comment:
A comment above each pass might be useful to explain each one

Reply:
Done